src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,216 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.*;
+import kitchenpos.infra.PurgomalumClient;
+import kitchenpos.testfixture.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+    MenuRepository menuRepository = new InMemoryMenuRepository();
+
+    MenuGroupRepository menuGroupRepository = new InMemoryMenuGroupRepository();
+
+
+    ProductRepository productRepository = new InMemoryProductRepository();
+
+    PurgomalumClient purgomalumClient = new FakePurgomalumClient();
+
+    MenuService menuService;
```

Comment:
클래스 내부에서만 사용되는 인스턴스 변수이니 접근제한자를 `priavte`로 지정해주시면 좋을 것 같습니다. 🙂
```suggestion
    private MenuRepository menuRepository = new InMemoryMenuRepository();

    private MenuGroupRepository menuGroupRepository = new InMemoryMenuGroupRepository();

    private ProductRepository productRepository = new InMemoryProductRepository();

    private PurgomalumClient purgomalumClient = new FakePurgomalumClient();

    private MenuService menuService;
```